src/components/TournamentFC.tsx

```diff
@@ -0,0 +1,84 @@
+import { useCallback, useEffect, useRef, useState } from 'react'
+
+import { usePocketBase } from '../contexts/PocketBaseContext'
+import type { Attendee } from '../models/Attendees'
+import type { Tournament } from '../models/Tournament'
+import AttendeeForm from './AttendeeForm'
+import { AttendeeTile } from './AttendeeTile'
+import './test.css'
+
+type Props = { data: Tournament }
+export const TournamentFC: React.FC<Props> = ({ data }: Props) => {
+  const [participants, setParticipants] = useState<Attendee[]>([])
+  const [suscribed, setSuscribed] = useState(false)
+  const { pb, user } = usePocketBase()
+
+  const dialog = useRef<null | HTMLDialogElement>(null)
+
+  useEffect(() => {
+    pb.collection('attendees')
+      .getFullList<Attendee>({
+        filter: `tournament = "${data.id}"`,
+      })
+      .then(res => setParticipants(res))
+  }, [data.id, pb, suscribed])
+
+  //Check si l'utilisateur est déjà inscrit
```

Comment:
```suggestion
  // check if user register
```
en anglais dans le code, en français dans l'appli et le routing